apps/coscrad-frontend/src/store/slices/resources/playlists/thunks.ts

```diff
@@ -0,0 +1,10 @@
+import { buildResourceFetchActionPrefix } from '../../utils/build-resource-fetch-action-prefix';
+import { createFetchThunk } from '../../utils/create-fetch-thunk';
+import { getApiResourcesBaseRoute } from '../shared';
+import { PLAYLISTS } from './constants';
+import { PlaylistIndexState } from './types';
+
+export const fetchPlaylists = createFetchThunk<PlaylistIndexState>(
+    buildResourceFetchActionPrefix(PLAYLISTS),
+    `${getApiResourcesBaseRoute()}/playlists`
```

Comment:
Fire up `npm run serve:all` and double check that via the network tab or redux devtools that the playlists are indeed coming through. I think this is the correct endpoint.